### PR TITLE
Extract: Add sId on models

### PR DESCRIPTION
**Pull request by PopDaph**
Adding `sId` fields for both models `EventSchema` and `ExtractedEvent` to enforce RESTfulness on API and consistency: 
- [x] Add new nullable field `sId`
- [x] Handle existing rows: create migration files to fill the sId.
- [x] New rows: set sId when creating a new object (both in app and in cli command).

Deployment: 
```
# On front
./admin/init-db.sh
env $(cat .env.local) npx tsx migrations/20230829_event_schema_sids.ts
env $(cat .env.local) npx tsx migrations/20230829_extracted_event_sids.ts
```

Next PRs: 
- Set sId field non nullable.
- Update API to use sIds.

### CODE REVIEW

File: front/admin/cli.ts [hunk 1]
```diff
@@ -567,7 +567,9 @@ const eventSchema = async (command: string, args: parseArgs.ParsedArgs) => {
       if (!args.properties) {
         throw new Error("Missing --properties argument");
       }
+      const sId = new_id();
       const schema = await EventSchema.create({
+        sId: sId.slice(0, 10),
```
Comment (fontanierh) [inline front/admin/cli.ts:572]: kind of nit, should we have a `size` optional param to this `new_id` function ? also, should it be `newId` instead ?

File: front/lib/models.ts [hunk 2]
```diff
@@ -1098,7 +1103,10 @@ EventSchema.init(
   {
     modelName: "event_schema",
     sequelize: front_sequelize,
-    indexes: [{ fields: ["workspaceId", "marker"], unique: true }],
+    indexes: [
+      { fields: ["workspaceId", "marker"], unique: true },
+      //{ fields: ["sId"], unique: true }, @todo daph add this after migration 20230829_extract_sids
```
Comment (fontanierh) [inline front/lib/models.ts:1108]: I think this could be switched on as is ? unique doesn't count for `NULL` I believe

**Review by fontanierh — DISMISSED** [review]

File: front/admin/cli.ts (hunk 1 above)
Comment (PopDaph) [reply front/admin/cli.ts:572]: Agreed about the NIT -> I even think it should not be a param but instead a `generateModelSid` that returns a 10 char sID, to enforce having always the same sId for models. I would do it in another PR so, as we need to refactor all places where we create an object that has an sID. OK for you?

Comment (fontanierh) [reply front/admin/cli.ts:572]: Yes of course !

Comment (PopDaph) [reply front/admin/cli.ts:572]: Done here: https://github.com/dust-tt/dust/pull/1162

**Pull request merged**